nBTC/sources/helper.move

```diff
@@ -0,0 +1,41 @@
+module nbtc::helper;
+
+use bitcoin_executor::utxo::{OutPoint, Data as UTXOData};
+use bitcoin_parser::encoding::u32_to_le_bytes;
+use bitcoin_parser::input;
+use bitcoin_parser::output;
+use bitcoin_parser::tx::{Transaction, new_unsign_segwit_tx};
+use nbtc::nbtc::NbtcContract;
+
+const DEFAULT_SEQUENCE: vector<u8> = x"ffffffff"; // no lock time, no Replace By Fee
+
+public fun compose_withdraw_unsign_tx(
```

Comment:
```suggestion
public fun compose_withdraw_tx(
```

Reply:
I think unsign this is more sense b/c we need sign this in the next state.